src/main/kotlin/org/jboss/tools/intellij/kubernetes/model/NamespaceProvider.kt

```diff
@@ -37,7 +37,7 @@ class NamespaceProvider(
         val provider = kindProviders[resource::class.java]
         var added = false
```

Comment:
why the logic is commented ? it will always return false from now ?

Reply:
good catch, missed this (generics are different in kotlin, so change wasn't obvious)